test/CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ function(tests folder)
   if (WIN32)
     LIST(REMOVE_ITEM TESTS "simd_op_check.cpp") # Relies on shell stuff that doesn't work on windows
   endif()
+  # TODO(srj): This test requires the runtime to be linked in as well, which 
```

Comment:
Is this comment still accurate? It looks like a holdover from when it was a correctness test